### DISCUSSION TITLE
Suggestion: Auto pin citation keys for different computers.

### DISCUSSION THREAD

**Discussion by earthmoon** [retorquere/zotero-better-bibtex#2365] [Ideas]
Hello,
As I use Zotero for two computers, each time I open Zotero from another computer, and fear that the Better Bibtex will change the citation keys. So I always pin the keys to avoid this kind of risk. But in my Windows 11 and Linux Debian 11. I see the citation key on two computers is different.
Thanks.

**Comment by retorquere** [comment]
https://retorque.re/zotero-better-bibtex/installation/preferences/#automatically-pin-citation-key-after

**Reply by earthmoon** [reply]
Hello,
May you tell me how to set this;

> When > 0, BBT will automatically pin the first citation keys it generates for an item after this many seconds.

I want to set it >0

**Reply by earthmoon** [reply]
> https://retorque.re/zotero-better-bibtex/installation/preferences/#automatically-pin-citation-key-after

Thanks for your help. I find it. I always ask silly questions. I hope you don't mind。
